src/solver/variable/bindConstraints.hxx

```diff
@@ -0,0 +1,616 @@
+                                                                   const Data::Area* area,
+                                                                   uint numSpace)
+{
+    /*
+    assert(NULL != area);
+    pAreas[area->index].template computeSpatialAggregateWith<SearchVCardT, O>(out, numSpace);
+    */
+}
+
+template<class NextT>
+template<class VCardToFindT>
+const double* BindingConstraints<NextT>::retrieveHourlyResultsForCurrentYear() const
+{
+    return nullptr;
+}
+
+template<class NextT>
+template<class VCardToFindT>
+inline void BindingConstraints<NextT>::retrieveResultsForArea(
+  typename Storage<VCardToFindT>::ResultsType** result,
+  const Data::Area* area)
+{
+    // pAreas[area->index].template retrieveResultsForArea<VCardToFindT>(result, area);
+}
+
+template<class NextT>
+template<class VCardToFindT>
+inline void BindingConstraints<NextT>::retrieveResultsForThermalCluster(
+  typename Storage<VCardToFindT>::ResultsType** result,
+  const Data::ThermalCluster* cluster)
+{
+    /*
+    pAreas[cluster->parentArea->index].template retrieveResultsForThermalCluster<VCardToFindT>(
+      result, cluster);
+    */
+}
+
+template<class NextT>
+template<class VCardToFindT>
+inline void BindingConstraints<NextT>::retrieveResultsForLink(
+  typename Storage<VCardToFindT>::ResultsType** result,
+  const Data::AreaLink* link)
+{
+    // pAreas[link->from->index].template retrieveResultsForLink<VCardToFindT>(result, link);
+}
+
+// ==============================================
+// ===========  Copy of area.inc.hxx  ===========
+// ==============================================
+
+template<class NextT>
+BindingConstraints<NextT>::~BindingConstraints()
+{
+    // Releasing the memory occupied by the binding constraints
+    if (pBindConstraints != nullptr)
+        delete[] pBindConstraints;
+}
+
+inline std::vector<std::pair<Data::BindingConstraintRTI*, uint>> getInequalityBindingConstraints(
+  Data::Study& study)
+{
+    std::vector<std::pair<Data::BindingConstraintRTI*, uint>> bindConstListToReturn;
+    Data::BindingConstraintRTI* allBindConst = study.runtime->bindingConstraint;
+    for (uint k = 0; k < study.runtime->bindingConstraintCount; k++)
+    {
+        // We pick inequality binding constraints only.
+        if (allBindConst[k].operatorType == '<' || allBindConst[k].operatorType == '>')
+        {
+            std::pair<Data::BindingConstraintRTI*, uint> bc_pair(&(allBindConst[k]), k);
+            bindConstListToReturn.push_back(bc_pair);
+        }
+    }
+    return bindConstListToReturn;
+}
+
+template<class NextT>
+void BindingConstraints<NextT>::initializeFromStudy(Data::Study& study)
+{
+    // This actually a vector of pairs where :
+    //  - first element of a pair is a pointer to a inequality binding constraint
+    //  - second element of a pair is the number of the previous BC in the runtime list of BCs.
+    std::vector<std::pair<Data::BindingConstraintRTI*, uint>> InequalityBCs
```

Comment:
If we want to filter constraints (only < or >), a vector of indices probably does the job here. Though I'm not sure we want to filter constraints. TBD with users & POs

Reply:
"a vector of indices probably does the job here" ==> done

Reply:
Where ?

Reply:
Finally done